tests/functional/test_resolve_jsxtag_gettext.js

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai';
+import * as babel from 'babel-core';
+import polyglotPlugin from 'src/plugin';
+import { rmDirSync } from 'src/utils';
+
+const translations = 'tests/fixtures/resolve_simple_gettext.po';
+
+const options = {
+    presets: ['es2015'],
+    plugins: [[polyglotPlugin, {
+        resolve: { translations },
+        discover: ['jt'],
+    }]],
+};
+
+describe('Resolve jsxtag-gettext', () => {
+    before(() => {
+        rmDirSync('debug');
+    });
+
+    it('should resolve simple gettext literal (without formatting)', () => {
+        const input = 'console.log(jt`simple string literal`);';
+        const result = babel.transform(input, options).code;
+        expect(result).to.contain('console.log(["simple string literal translated"]);');
```

Comment:
In these tests I modified the original `tag_gettext` functional tests to expect an array output.